src/tbb/misc_ex.cpp

```diff
@@ -63,7 +65,7 @@ static const dynamic_link_descriptor iompLinkTable[] = {
 #endif
 
 static void set_thread_affinity_mask( std::size_t maskSize, const basic_mask_t* threadMask ) {
-#if __linux__
+#if __unix__
     if( sched_setaffinity( 0, maskSize, threadMask ) )
 #else /* FreeBSD */
     if( cpuset_setaffinity( CPU_LEVEL_WHICH, CPU_WHICH_TID, -1, maskSize, threadMask ) )
```

Comment:
```suggestion
#if __FreeBSD__ || __NetBSD__ || __OpenBSD__
    if( sched_setaffinity( 0, maskSize, threadMask ) )
#else /* __unix__ */
    if( cpuset_setaffinity( CPU_LEVEL_WHICH, CPU_WHICH_TID, -1, maskSize, threadMask ) )
```

Reply:
The internal CI found that my suggestion was incorrect. Lines 70 and line 72 are also should be swapped...

Reply:
Your suggestion is ok. I forgot to change one thing in the first CI try. I fixed it later and the last CI run is succesful.